docs/guides/src/main/server/enabletls.adoc

```diff
@@ -0,0 +1,58 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+
+<@tmpl.guide
+title="Configure Keycloak to use TLS"
```

Comment:
I'd probably avoid `Keycloak` in titles, as it's kinda obvious isn't it? Can leave this as is for now, then we can review this when we have a bit more guides in, and have a follow-up PR to review all titles separately.

Reply:
+1 for the second iteration for polishing.

Reply:
will change it nevertheless while i'm on it